src/IssueViz/Editor/QuickActions/QuickFixes/QuickFixActionsSource.cs

```diff
@@ -71,11 +70,13 @@ internal sealed class QuickFixActionsSource : ISuggestedActionsSource
         {
             var allActions = new List<ISuggestedAction>();
 
-            if (IsOnIssueWithQuickFixes(range, out var issuesWithFixes))
+            if (IsOnIssueWithApplicableQuickFixes(range, out var issuesWithFixes))
```

Comment:
IsOnIssueWithApplicableQuickFixes returns true if the quick fix has at least 1 applicable fix: this would be used by VS to know if it should even render the lightbulb. Then `applicableFixes` are filtered to create an action only for the applicable fixes.